credsweeper/rules/config.yaml

```diff
@@ -174,7 +174,7 @@
   severity: high
   type: pattern
   values:
-    - (?P<value>(?i)heroku(.{0,20})?[0-9a-f]{8}(-[0-9a-f]{4})+-[0-9a-f]{12})
+    - (?P<value>(?i)heroku(.{0,20})?[0-9a-f]{8}(-[0-9a-f]{4})+-[0-9a-f]{12})([^0-9a-z]|$)
   filter_type: GeneralPattern
   use_ml: true
```

Comment:
Could you explain me why this right value is different with others?

- This one : `([^0-9a-z]|$)`
- Others : `([^=0-9A-Za-z_/+-]|$)`

Reply:
Something were missed. I'll check